### PR TITLE
Set logging level on build

### PR DESCRIPTION
**Pull request by hyunsun**
Also fixed the order of logging levels.

### CODE REVIEW

**Comment by omecproject** [comment]
Can one of the admins verify this patch?

**Comment by ghost** [comment]
@zdw do you know why this patch did not trigger off the jenkins job?

**Comment by zdw** [comment]
retest this please

**Comment by zdw** [comment]
> @zdw do you know why this patch did not trigger off the jenkins job?

The Jenkins trigger only runs after a patchset is approved by a user who is a member of the `omec-project` group.  The trigger phrases to include in the comment are here: https://github.com/jenkinsci/ghprb-plugin

There seems to be a problem with the system running the tests.

File: src/common/log.c [hunk 1]
```diff
@@ -23,21 +23,21 @@
...
+#endif
+
+static const char *log_level_name[] = { "DEBUG", "INFO", "WARN", "ERROR" };
 
 void log_message(int l, const char *file, int line, const char *fmt, ...)
 {
 	va_list arg;
-	if (g_nolog) return;
-	if(g_log_level > l) return;
+	if(LOG_LEVEL > l) return;
```
Comment (ghost) [inline src/common/log.c:36]: For the No log case.
if #define LOG_LEVEL LOG_NEVER
if log_message(LOG_NEVER,....)
look still log the message.

Comment (hyunsun) [reply src/common/log.c:36]: You're not supposed to use LOG_NEVER in your "log_message" function.
If you don't want logging, just set LOG_LEVEL to LOG_NEVER then all valid log levels, from LOG_DEBUG to LOG_ERROR, will be ignored.

Comment (ghost) [reply src/common/log.c:36]: yes, make sense

**Comment by micpez** [comment]
retest this please

**Comment by micpez** [comment]
retest this please

**Comment by micpez** [comment]
retest this please

**Comment by micpez** [comment]
retest this please

**Comment by ghost** [comment]
+1

**Comment by ghost** [comment]
@vbarate can give a review for this patch?

**Comment by hyunsun** [comment]
@badhrinathpa Could you review this patch?

**Pull request closed without merging**